recipes/nasm/all/conanfile.py

```diff
@@ -92,13 +89,12 @@ def package(self):
                 shutil.copy2("ndisasm.exe", "ndisasmw.exe")
         else:
             autotools = Autotools(self)
-            # TODO: replace by autotools.install() once https://github.com/conan-io/conan/issues/12153 fixed
-            autotools.install(args=[f"DESTDIR={unix_path(self, self.package_folder)}"])
+            autotools.install()
             rmdir(self, os.path.join(self.package_folder, "share"))
 
     def package_info(self):
         self.cpp_info.libdirs = []
         self.cpp_info.includedirs = []
 
         # TODO: Legacy, to be removed on Conan 2.0
-        self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
```

Comment:
must not be removed

Reply:
Hi @SpaceIm,

I thought:

> self.runenv_info.append_path("PATH", os.path.join(self.package_folder, "bin"))

was the Conan 2 compatible way fo saying the same thing. No?

Reply:
Restored now that Conan 2.0 doesn't error out on this construct.